graphql/util.go

```diff
@@ -6,7 +6,7 @@ package graphql
 // NoUnmarshalJSON is intended for the use of genqlient's generated code only.
 //
 // It is used to prevent a struct type from inheriting its embed's
-// UnmarshalJSON method: given a type
+// UnmarshalJSON method so for a given type:
```

Comment:
This doesn't flow as well into the continuation below, I think it's better as-is.  Maybe even better would be:
```suggestion
// UnmarshalJSON method, so if we construct a type:
```
and/or rephrase the subsequent few lines.